docs/parser_plugins.md

```diff
@@ -1 +1,210 @@
+Parsers _automate the conversion_ of a file format common to a specific piece of software or a community into the NOMAD format.
+The schema for computational techniques was introduced in the previous section.
+
+The key advantages of the NOMAD schema are summed up in **FAIR**mat's core values:
+
+- **F**indable: a wide selection of the extracted data is indexed in a database, powering a the search with highly customizable queries and modular search parameters.
+- **A**ccessible: the same database specifies clear API and GUI protocols on how retrieve the _full_ data extracted.
+- **I**nteroperable: we have a diverse team of experts who interface with various materials science communities, looking into harmonizing data representations and insights among them. Following the NOMAD standard also opens up the (meta)data to the NOMAD apps eco-system.
+- **R**eproducible: data is not standalone, but has a history, a vision, a workflow behind it. Our schema aims to capture the full context necessary for understanding and even regenerating via metadata.
+
+## Modular Design and Plugins
+
+Historically, our parsers have come always come together as a neat bundle.
+As interest in research data management has become more widespread and recognized across scientific disciplines, NOMAD has adapted along with those interests.
+Nowadays, you can find besides the [central NOMAD cloud service](), also standalone NOMAD Oasis deployments geared towards the needs of individual institutes or research groups.
+
+As such, NOMAD is shifting to a _plugin model_, where each deployment administrator can customize their setup.
+This permits a light-weight base installation and effective data processing pipelines.
+Conversely, it also lowers the threshold for whomever wants to contribute to this open-source community: you can focus on just a single schema or parser.
+The possibilities are as wide as you can make them, we just facilitate the common basis.
+
+In the rest of this page, we are going to take you on a birds-eye overview of how a parser developer approaches these plugins.
+
+### Getting started
+
+Getting hands-on with a plugin is just as easy visiting our [GitHub template]() and _using one in your own codespace_.
+The template will appear almost empty at the start, but following the instructions `README.md`  and during the `cruft` setup will allow you to tune the project to your needs.
+Just a couple of notes on the latter:
+
+- NOMAD uses the Apache 2.0 license. Unless your organization prohibits this, please select the same license for maximal legal compatibility.
+- Parsers require both a `parsers` and a `schema_packages` folder. The reason for the second folder will soon become apparent.
+- We use generically named files like `parsers/myparser.py` as blueprints by copying them to new, more clearly named files. Each file format, its own Python file.
+
+### Managing Entry Points
```

Comment:
> Also, the last point on the nomad.yaml goes beyond the scope

Meanwhile, @JFRudzinski is asking me to elaborate this point. I'll link the docs, though.

> Why would you explain the whole functionality of EntryPoints here?

Idt that it was one to avoid circular imports, but nice. Regardless, this is now the system by which any plugin is exposed (by definition). NOMAD requires it to pick up on them. In the end, that's all the audience needs to know.
(Yes, you can add certain dependencies as optional to the `pyproject.toml`, but to my understanding this does not apply to plugins.)

Having recently gone over the entry points, our docs were quite confusing (will give some feedback to Area D later). I definitely think that there's merit in explaning the structure, both textually and visually.
With that said, I'll hide the more in-depth discussion via an admonishment.

Reply:
Migrated most of the entry point discussion to a section Extra at the bottom.
I retain 1 short paragraph in "Getting Started".